src/js/lotto/lottoDomain.js

```diff
@@ -0,0 +1,42 @@
+import { $, WARNING_WHEN_NOT_IN_1000_UNITS } from "../shared/consts.js"
+import { removeCssToggleWhenResubmitting } from "./util/removeCssToggleWhenResubmitting.js";
+
+import lottoTicket from "./lottoTicket.js"
+
+const $lottoPurchaseInput = $('.lotto-purchase-input');
+const tagsShownAccordingToPurchaseStatus = [$('#purchased-lottos'), $('#lotto-winning-numbers-form')]
+
+const checkDisplayWhenResubmitting = () => {
+  tagsShownAccordingToPurchaseStatus.map(tag => { 
+    if (tag.style.display === "block") tag.style.display = 'none'
+  })
+}
+
+const passValidatitonLottoPurchaseAmount = amount => {
+  amount = Number(amount)
+
+  if (amount < 1000 || amount >= 1000000) {
+    checkDisplayWhenResubmitting()
+    return false
+  }
+
+  if (amount % 1000 !== 0) {
+    $lottoPurchaseInput.value = ""
+
+    alert(WARNING_WHEN_NOT_IN_1000_UNITS)
+    checkDisplayWhenResubmitting()
+    return false
+  }
+
+  return true
+}
+
+const lottoDomain = amount => {
+  if (passValidatitonLottoPurchaseAmount(amount)) {
+    tagsShownAccordingToPurchaseStatus.map(tag => tag.style.display = "block")
```

Comment:
```suggestion
    tagsShownAccordingToPurchaseStatus.forEach(tag => tag.style.display = "block")
```
map은 요소를 반복시키면서 새로운 값을 반환하는 메소드입니다.
그래서 이럴 땐 forEach를 사용하는게 좋답니다!

Reply:
수정하겠습니다!